### ISSUE TITLE
"The guest machine entered an invalid state while waiting for it to boot. Valid states are 'starting, running'. The machine is in the 'poweroff' state. Please verify everything is configured properly and try again."

### ISSUE THREAD

**Issue by roelvanduijnhoven** [hashicorp/vagrant#2157]
At the moment I can no longer `vagrant up` from an initial `Vagrentfile` using box `precise64` (but also `precise32`). The strange thing is that it did work before (just after I installed it).

The output of running `VAGRANT_LOG=info vagrant up` is found in this gist: https://gist.github.com/roelvanduijnhoven/6477492

I have no clue what is going wrong!

The error I get is:

```
"The guest machine entered an invalid state while waiting for it to boot. Valid states are 'starting, running'. The machine is in the 'poweroff' state. Please verify everything is configured properly and try again."
```

**Comment by adamsynnott** [comment]
I had the same issue, updating Virtual Box fixed the issue for me - version 4.3.20

**Comment by adamhoke** [comment]
@eriksencostsa 's solution worked for me:
vb.customize ["modifyvm", :id, "--accelerate3d", "off"]

I was using OSX Yosemite host, CentOS 5.5 Guest

**Comment by jirawute** [comment]
test

**Comment by rubo77** [comment]
@roelvanduijnhoven You said

https://github.com/mitchellh/vagrant/issues/2157#issuecomment-24017403
> After I tried this I manually removed all three images and added a new precise32 box again. Now it works again.. 

can you tell us the exact line you added in your Vagrantfile?

**Comment by roelvanduijnhoven** [comment]
No clue anymore @rubo77. More than a year ago!

**Comment by ahummel25** [comment]
Is anyone here familiar with how to port forward your vagrant VM so that you can successfully fire up OpenBazaar on it?

I keep thinking I have it right but I keep getting these errors every time.

![image](https://cloud.githubusercontent.com/assets/12643871/7947164/f737a884-0940-11e5-9357-3a3e753be7f0.png)

**Comment by Swordsman-Inaction** [comment]
If you are using VirtualBox, install Extension Pack might be help.

**Comment by cdaringe** [comment]
updating to el capitan killed it!  thanks for the tips!  booting into the gui yielded "Kernel driver not installed (rc=-1908)".  googling that error led to several forums indicating that virtualbox has kernel hooks that were incompatible after the OS update. reinstalling virtualbox solved the issue!

**Comment by switchflip** [comment]
I also update from vagrant 4.3.1 to 5 and everything works swimmingly :+1:

**Comment by MrLokans** [comment]
Same problem occured when I tried to specify 'config.vm.network "private_network", type: "dhcp"' setting. I solved that issue by loading kernel modules for the virtual box:
```bash
# modprobe vboxnetflt
# modprobe vboxpci
# modprobe vboxnetadp
```
Don't forget to check whether they were truly loaded by
```bash
lsmod | grep vbox
```

**Comment by KokareIITP** [comment]
I am working with Vagrant 1.7.4 and Virtual Box 5.0.10 , but when I do vagrant up with git bash I get the following error,
![screenshot 19](https://cloud.githubusercontent.com/assets/9394028/11532205/63de22ca-9928-11e5-84f2-81bb0e7b8db1.png)

How do I resolve this ?

**Comment by yurivyatkin** [comment]
**TL;DR** Go to "VirtualBox Host-Only Network Properties" and disable and then enable "VirtualBox NDIS6 Bridged Networking Driver".

I just got started with Vagrant under Windows 10, and immediately run into this issue. Following comments above, I tried to start the vagrant's machine manually in VirtualBox GUI, but it failed with the error message "Failed to open/create the internal network 'HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter' (VERR_INTNET_FLT_IF_NOT_FOUND)." Researching on the latter input, I found this mysterious solution http://stackoverflow.com/a/33733454/1008341

**Comment by set0x** [comment]
Windows: 10, Virtual Box,: 5.0.10, Vagrant: 1.7.4... Solution

Went to the C:\Program Files\Oracle\VirtualBox\drivers\vboxdrv directory, right clicked on VBoxDrv.inf and selected Install. I then went back to my console and typed 'sc start vboxdrv'.

https://forums.virtualbox.org/viewtopic.php?f=6&t=66442

**Comment by junluoyui** [comment]
How to go to VirtualBox Host-Only Network Properties?

**Comment by set0x** [comment]
![tut](https://cloud.githubusercontent.com/assets/15133015/11826677/32d5d79a-a387-11e5-8f15-76bf39341762.png)

**Comment by junluoyui** [comment]
Thanks a lot~

**Comment by milan612** [comment]
Using Ubuntu Trusty on OSX. checked 
"Enable hyperversion applications on this virtual machine." in VM setting and all good :)

**Comment by sgsheg** [comment]
:+1:  It help me.

**Comment by miparnisari** [comment]
I got this error right after upgrading Windows to Windows Anniversary Update.

**Comment by mattkaye** [comment]
Just found this:
http://superuser.com/questions/1111027/fix-vagrant-up-after-windows-10-anniversary-update

**Comment by miparnisari** [comment]
Yep, that answer did the trick for me.

**Comment by thiyagarajan** [comment]
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'hashicorp/precise64' is up to date...
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 3000 (guest) => 3000 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

If you look above, you should be able to see the error(s) that
Vagrant had when attempting to connect to the machine. These errors
are usually good hints as to what may be wrong.

If you're using a custom box, make sure that networking is properly
working and you're able to connect to the machine. It is a common
problem that networking isn't setup properly in these boxes.
Verify that authentication configurations are also setup properly,
as well.

If the box appears to be booting properly, you may want to increase
the timeout ("config.vm.boot_timeout") value.

VTR@VTR-pc MINGW64 ~

**Comment by mattkaye** [comment]
thiyagarajan, this has already been solved. It has nothing to do with networking or a timeout issue

**Comment by UniFreak** [comment]
using vagrant on Windows 7 64bit, with Virtual box 4.3.12. every thing is normal when I first installed vagrant and some box. then this day it just `vagrant up` just can start, with the same erorr message.

Vt-* is enabled. other box in virtual machine is fine.

I solved this by update virtual box to 5.1.10

**Comment by CatoMinor** [comment]
I received the "The guest machine entered an invalid state while waiting for it to boot" error after performing a reinstall of Norton Internet Security (antivirus software).  I was able to run my VM via the VirtualBox GUI, just not with `vagrant up` (also MySQL Workbench could not connect to DB).  I went into my BIOS, *disabled* virtualization, restarted, went into BIOS again and then *re-enabled* virtualization.  Odd as it may seem, following this, `vagrant up` (and MySQL Workbench) began working properly again.

**Comment by agnesdai** [comment]
I had the error. I followed the instructions here and dowloaded virtual box 4.3.12  ; vagrant init hashicorp/precise32 ; vagrant up and vagrant ssh and it worked in less than 5 mins. Hope this helps.

**Comment by starikovs** [comment]
Hey! I just had this error when I upgraded my osx to macOS Sierra:

```
The guest machine entered an invalid state while waiting for it
to boot. Valid states are 'starting, running'. The machine is in the
'aborted' state. Please verify everything is configured
properly and try again.

If the provider you're using has a GUI that comes with it,
it is often helpful to open that and watch the machine, since the
GUI often has more helpful error messages than Vagrant can retrieve.
For example, if you're using VirtualBox, run `vagrant up` while the
VirtualBox GUI is open.
```
So, after upgrading the VirtualBox from 4.3 to the current 5.1 version, and vagrant from 1.7.2 to 1.9.5, it fixed!!

**Comment by Alendorff** [comment]
Virtual Box Version 5.1.22 
Vagrant 1.9.5
Windows 10

Have this problem too.

**Comment by lzh1104** [comment]
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: You are trying to forward to privileged ports (ports <= 1024). Most

==> default: operating systems restrict this to only privileged process (typical
ly
==> default: processes running as an administrative user). This is a warning in
case
==> default: the port forwarding doesn't work. If any problems occur, please try
 a
==> default: port higher than 1024.
==> default: Forwarding ports...
    default: 22 (guest) => 1322 (host) (adapter 1)
    default: 80 (guest) => 80 (host) (adapter 1)
    default: 443 (guest) => 443 (host) (adapter 1)
    default: 3306 (guest) => 3306 (host) (adapter 1)
    default: 6379 (guest) => 6379 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
The guest machine entered an invalid state while waiting for it
to boot. Valid states are 'starting, running'. The machine is in the
'unknown' state. Please verify everything is configured
properly and try again.

If the provider you're using has a GUI that comes with it,
it is often helpful to open that and watch the machine, since the
GUI often has more helpful error messages than Vagrant can retrieve.
For example, if you're using VirtualBox, run `vagrant up` while the
VirtualBox GUI is open.

The primary issue for this error is that the provider you're using
is not properly configured. This is very rarely a Vagrant issue.

**Comment by MehdiElayeb** [comment]
ayebm vagrant
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'bento/ubuntu-16.04-i386'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'bento/ubuntu-16.04-i386' is up to date...
==> default: Setting the name of the VM: vagrant_default_1519659828987_21771
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 8000 (guest) => 8000 (host) (adapter 1)
    default: 8080 (guest) => 8080 (host) (adapter 1)
    default: 5000 (guest) => 5000 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
The guest machine entered an invalid state while waiting for it
to boot. Valid states are 'starting, running'. The machine is in the
'unknown' state. Please verify everything is configured
properly and try again.

If the provider you're using has a GUI that comes with it,
it is often helpful to open that and watch the machine, since the
GUI often has more helpful error messages than Vagrant can retrieve.
For example, if you're using VirtualBox, run `vagrant up` while the
VirtualBox GUI is open.

The primary issue for this error is that the provider you're using
is not properly configured. This is very rarely a Vagrant issue.

**Comment by guilhermeureis** [comment]
Step by step:
I opened Virtualbox.
I turned off the Homestead
Then I used the vagrant up

It worked!

**Comment by godhandkiller** [comment]
@guilhermeureis Thank you!

First I destroy the vagrant machine, so I created a new one (I'm using Laravel Homestead).
Then I followed @guilhermeureis steps and It worked!

**Comment by oLDu67** [comment]
Thank you @guilhermeureis
